src/modules/product-category/entities/product-category.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { Entity, PrimaryGeneratedColumn, Column, Index, OneToMany } from 'typeorm';
+import { ApiProperty } from '@nestjs/swagger';
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { Product } from '../../products/entities/product.entity';
+@Entity('categories')
+export class ProductCategory extends AbstractBaseEntity {
```

Comment:
Extending AbstractBaseEntity does not require you to create a uuid column any more. Please remove